prisma/schema.prisma

```diff
@@ -207,3 +209,19 @@ model ExerciseSubmission {
 
   @@map("exerciseSubmissions")
 }
+
+model Discussion {
```

Comment:
I'm a little bit confused by naming this table `Discussion`. Wouldn't `Comment` make more sense?

Reply:
I feel `Discussion` refer to the feature whilst a `Comment` refer to the comments in the `Discussion`.

Reply:
![image](https://user-images.githubusercontent.com/77421872/194374732-0c07db07-1a33-4cfa-b515-204528bba271.png)

Open to suggestions about naming it something different but there's already a `Comment` table.

Reply:
Lets finalize all these details before implementation. We should not have to be touching this model in the backend anymore to finish implementing a MVP for the discussions flow

Reply:
> Lets finalize all these details before implementation. After implementing this model, we should not have to be touching this model in the backend anymore to finish implementing a MVP for the discussions flow

What do you mean by "finalize"?

Reply:
If we are following a design doc, all of the architecture should be decided upon before implementation. By finalized details I mean complete design doc. What a complete design doc looks like will differ depending on what feature the design doc was made for. Here are good tips on design docs, it's also in our engineering practices
https://www.industrialempathy.com/posts/design-docs-at-google/